analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestAnalysisComparer_2.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class SecondClass
+    {
+        async void WrongLength() { } // Noncompliant {{Return 'Task' instead.}}
+        //    ^^^^^
+
+        void Default()// Secondary [flow-0]
```

Comment:
The first test case file explains each problem with a method name. Here I'm struggling with some. Can you rename them in a similar way as they are named in the first file?